README.md

```diff
@@ -93,6 +93,54 @@ The build process uses a Node.js container image to build all assets, and copy t
   
 `export/` can then be packaged to be released on Github, or reused elsewhere.
 
+
+### Styling
+
+#### CSS Custom Properties
+These guidelines govern the use of custom properties in this codebase:
+
+1. global, constant values should be defined in scss, there is no need to express these as custom properties
+1. use a custom property when a css property changes e.g. in response to DOM conditions (e.g. `:focus`), media queries or via JavaScript
+1. global values (i.e. those set on `:root`) should be named in uppercase to indicate they're global. (Aside: as we're not doing any theming, there shouldn't be many of these.)
```

Comment:
Who else does this? Not sure what the reasoning is...

Reply:
This is advice from Mike Riethmuller in Chapter 4 ("A Strategic Guide To CSS Custom Properties") of Smashing Book 6. The point is that as CSS custom properties are case sensitive, and most are likely to be local, capitalising the global ones makes them easy to identify. Always good to know if you're altering a global value!

Reply:
Probably need to try it out more to see the benefit!